pkg/controllers/dnshealthcheckprobe/notifier.go

```diff
@@ -0,0 +1,51 @@
+package dnshealthcheckprobe
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/apis/v1alpha1"
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/health"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
nit: grouping and order of imports is inconsistent with others.

Reply:
fixed.